src/commands/watch.rs

```diff
@@ -132,8 +132,14 @@ impl WatchCommand {
         let source_code_tx = Arc::new(source_code_tx);
         let manifest_tx = Arc::new(manifest_tx);
 
+        // Check if the --direct-mounts flag is set We can skip watching assets
```

Comment:
Nit: the first part of the comment restates the code - you can probably simplify the comment to e.g. "No need to restart on asset changes if direct-mounting"

Reply:
Done.